game/plugins/src/main/kotlin/gg/rsmod/plugins/content/skills/slayer/slayer_masters_dialogue.plugin.kts

```diff
@@ -16,6 +14,14 @@ masters.forEach { npcId ->
 
     val slayerMaster = SlayerMaster.values().firstOrNull { it.id == npcId } ?: return@forEach
 
+    /**
+    on_npc_option(npc = Npcs.VANNAKA, option = "get-task") {
+        player.queue {
+            giveTask(this, slayerMaster)
+        }
+    }**/
+
+
     on_npc_option(npc = npcId, option = "talk-to") {
```

Comment:
We should reference the SlayerMaster enum and iterate through all the masters instead of constructing a new list.

Reply:
Nice catch! Using the existing enums class will have of time complexity of O(1), while the list is O(n).